Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// The BlockStoreLoop simultaneously finds and downloads blocks
+    /// and stores them in the BlockRepository
+    /// <see cref="DownloadAndStoreBlocks"/>
```

Comment:
`<see>` is for inline reference - like you have "The BlockStoreLoop simultaneously finds" <- there you could use `<see>` for BlockStoreLoop.

If you want to tell user to also see something else, then `<seealso>` (and that is used outside `<summary>`)